representatives/models.py

```diff
@@ -130,6 +125,31 @@ def gender_as_str(self):
         genders = {0: 'N/A', 1: 'F', 2: 'M'}
         return genders[self.gender]
 
+    def get_country(self):
+        """
+        Get the representative country.
+
+        Note that this will butcher your database if you didn't get the
+        representatives from ``with_countries()``.
+        """
+        for m in self.mandates.all():
+            if not m.constituency.country_id:
+                continue
+
+            return m.constituency.country
+
+    def get_current_mandate(self):
+        today = date.today()
+
+        for m in self.mandates.all():
+            if m.end_date < today:
+                continue
+
+            if m.group.kind != 'group':
+                continue
+
+            return m
+
```

Comment:
Those two functions can be fully done in SQL no? like self.mandates.filter(consitituency__country_id__isnull=False)[0] and self.mandates.filter(end_date__lt=date.today(), group__kind="group")[0] or am I missing something?